.vscode/cspell.json

```diff
@@ -259,6 +259,8 @@
         "WINDOWSPOOL",
         "WINDOWSVMIMAGE",
         "winget",
-        "Xunit"
+        "Xunit",
+        "azuretools",
```

Comment:
Looks like other entries are sorted alphabetically in this list. Do we want to continue that?